src/Shell/Task/ModelTask.php

```diff
@@ -560,7 +560,9 @@ public function getEntityPropertySchema(Table $model)
 
             $properties[$column] = [
                 'kind' => 'column',
-            ] + $columnSchema;
+                'type' => $columnSchema['type'],
+                'null' => $columnSchema['null'],
+            ];
```

Comment:
Why not passing all?

Reply:
We don't need all the properties right now, and it creates a wider interface than we need with more chances to break behavior.